src/mixins/java/org/spongepowered/common/mixin/core/world/entity/ai/goal/GoalMixin.java

```diff
@@ -42,6 +43,8 @@
 @Mixin(Goal.class)
 public abstract class GoalMixin implements GoalBridge {
 
+    @Shadow protected abstract int adjustedTickDelay(int $$0);
```

Comment:
I'd recommend explicitly naming the method shadowed `shadow$adjustedTickDelay` so it's clearer